src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationBinderTests.cs

```diff
@@ -505,6 +505,114 @@ public enum TestSettingsEnum
             Option2,
         }
 
+        public class WhenBindingCollectionsWeCanControlExceptionsWithTheErrorOnUnknownConfigurationProperty
```

Comment:
I know this doesn't fit in with the existing naming convention, but I was struggling to find suitably terse named to describe the scenario **and** the outcome.  Suggestions very welcome.